src/components/SanitizeHtml/PSanitizeHtml.vue

```diff
@@ -10,17 +10,18 @@
   import { PUnwrap } from '@/components/Unwrap'
 
   const props = defineProps<{
-    html: string,
+    html: string | HTMLElement | DocumentFragment,
+    config?: dompurify.Config,
   }>()
 
   const forbiddenAttrs = ['style', 'action', 'method', 'onclick', 'onmouseover', 'onload', 'data', 'onmousedown', 'onmouseup']
   const forbiddenTags = ['script', 'style', 'iframe', 'form']
   const useProfiles = { svg: true, html: true }
 
-  const sanitize = (text: string): string => {
+  const sanitize = (text: string | HTMLElement | DocumentFragment): string | HTMLElement | DocumentFragment => {
     return dompurify.sanitize(
       text,
-      {
+      props.config ?? {
```

Comment:
nit: this would be nice rather than declaring the default inline here. Could probably combine with the consts above and just declare a single const with the correct type. 
```
props.config ?? defaultConfig
```

Reply:
Ah shoot I merged before this comment came through, sorry!

Reply:
No that's my bad, I accidentally submitted the review without this comment and had to go back and add it.

Reply:
https://github.com/PrefectHQ/prefect-design/pull/1013